Jenkinsfile

```diff
@@ -39,8 +39,8 @@ def postEmulator(String coverageNameAndLogcatPrefix) {
     archiveArtifacts "${coverageNameAndLogcatPrefix}_logcat.txt"
 }
 
-def useWebTestParameter() {
-    return env.USE_WEB_TEST?.toBoolean() ? '-PuseWebTest' : ''
+def webTestUrlParameter() {
+    return env.WEB_TEST_URL?.isEmpty() ? '' : "-Pweb_test_url='${params.WEB_TEST_URL}'"
```

Comment:
for -P I think camel case is normally used for naming and not underline so it would be -PwebTestUrl like before the -PuseWebTest